cmd/network/verification.go

```diff
@@ -433,7 +433,7 @@ func (e *EgressVerification) getSubnetIds(ctx context.Context) ([]string, error)
 		}
 
 		if len(resp.Subnets) == 0 {
-			return nil, fmt.Errorf("found 0 subnets with kubernetes.io/cluster/%s=owned and %s, consider the --subnet-id flag", e.cluster.InfraID(), e.cluster.InfraID())
+			return nil, fmt.Errorf("found 0 subnets with kubernetes.io/cluster/%s and %s, consider the --subnet-id flag", e.cluster.InfraID(), e.cluster.InfraID())
```

Comment:
```suggestion
			return nil, fmt.Errorf("found 0 subnets with tags: kubernetes.io/cluster/%s and %s, consider the --subnet-id flag", e.cluster.InfraID(), nonByovpcPrivateSubnetTagKey)
```
I'm not sure that this current error message makes sense